src/layouts/sculpt.js

```diff
@@ -91,23 +91,26 @@ const Maker = (props) => {
                   alt={`${maker.name} - ${sculpt.name} - ${c.name}`}
                 />
               </Link>
-              <div className="font-bold flex flex-row pt-3 px-2 relative">
+              <div className="font-bold flex justify-between items-center pt-3 px-2 relative">
```

Comment:
Not sure about those classes. Look on Nigthcaps bombkings. It's more homogenous without it